File: hw/acuña/u2/HW18ZooInPythonJavaScript/zoo/ec.edu.espe.zoo.model/Primate.js
const {Mammal} = require('./Mammal.js');

class Primate extends Mammal {
    constructor(id, name, birthDate, gender, numberOfLegs, cage) {
        super(id, name, birthDate, gender, numberOfLegs, cage);
    }
}

module.exports = { Primate };

File: ws/arboleda/u2/WS24Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Primate.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Abner Arboleda, Logic Legion, DCCO-ESPE
 */
public abstract class Primate extends Mammal{

    public Primate(int id, String description, Date dateOnborn, char gender, int numberOfBones, Cage cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

}
